### PR TITLE
LWR-83: publication/asset support including hanger bg variant

### PR DESCRIPTION
**Pull request by melwong-jcc**
Added:
- Hanger background control
- List molecule
- AssetBundle molecule as content template for Publications, potentially for other media types like JCC forms
- Asset organism (section example) to create a section that combines List and AssetBundle. Keeping Hero and Hanger separate for flexibility at the Drupal level for now but can build a page if needed.

### CODE REVIEW

**Review by sleepingmonk — COMMENTED** [review]
@melwong-jcc  There are quite a few details here.  Please let me know if anything doesn't make sense.

**Review by sleepingmonk — COMMENTED** [review]
@melwong-jcc  There are quite a few details here.  Please let me know if anything doesn't make sense.

File: src/stories/Molecules/List/List.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.list {}
+
+.list__container {
+  padding: var(--s1);
```
[the remark this answers is not in the document]
Comment (melwong-jcc) [reply src/stories/Molecules/List/List.css:4]: This was done to align the sidebar list being used to display metadata with the AssetBundle content which also has padding. It's probably why originally I was looking into add margin as an option which perhaps we should do to add .box or something optionally.

Without the padding also, the sidebar content and the main content will have zero spacing between each other.

File: src/stories/Molecules/AssetBundle/AssetBundle.twig [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+{# Include variant classes form above. #}
+{% set classes = [
+  'asset-bundle'
+] %}
+
+<div {{ attributes.addClass(classes) }}>
+  {% if text %}<div class="asset-bundle__body">{{ text }}</div>{% endif %}
+  {% if assets or type %}
+    <div class="asset-bundle__details">
+      {{ primary_file_type }}
```
[the remark this answers is not in the document]
Comment (melwong-jcc) [reply src/stories/Molecules/AssetBundle/AssetBundle.twig:14]: No need to get complicated. Display only if the first asset exists else there is not type to display.

File: src/stories/Molecules/AssetBundle/readme.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<!-- This is the general documentation layout. Add or remove any sections as needed, but try to stay consistent across components. -->
+# AssetBundle
+
+Standard display for an asset page including general documents, publications, and potentially video & audio.
+
+[//]: # (<details>)
```
[the remark this answers is not in the document]
Comment (melwong-jcc) [reply src/stories/Molecules/AssetBundle/readme.md:6]: That's how you comment out readme content but removed completely as suggested.

**Pull request closed without merging**